tools/SetupFlow/DevHome.SetupFlow/Views/DevDriveView.xaml

```diff
@@ -21,7 +21,7 @@
         VerticalScrollBarVisibility="Auto"
         HorizontalScrollBarVisibility="Disabled"
         Name="Scroller"
-        Margin="10 0 10 20"
+        Margin="10 0 0 20"
         Canvas.ZIndex="1"
         MaxWidth="{ThemeResource MaxPageContentWidth}"
         HorizontalContentAlignment="Center"

```

Comment:
```suggestion
        HorizontalScrollBarVisibility="Disabled"
        Name="Scroller"
        Canvas.ZIndex="1"
        HorizontalContentAlignment="Center"
```

then move the deleted lines down into the stackpanel on line 30. This should make sure the scrollviewer margins are left as default (left side of window) and the content within it sets its own margins. So the StackPanel on line 30 will look like this:

```xaml

    <StackPanel
        Margin="20 0 20 20"
        MaxWidth="{ThemeResource MaxPageContentWidth}">
```

Reply:
Done.  I believe I did what you were thinking.  Double check to make sure I am right.